src/orbit-client/src/main/kotlin/orbit/client/OrbitClient.kt

```diff
@@ -126,7 +128,9 @@ class OrbitClient(val config: OrbitClientConfig = OrbitClientConfig()) {
             }
 
             // Get first lease
-            nodeLeaser.joinCluster()
+            retry(retryDelay = Duration.ofSeconds(1)) {
```

Comment:
Should this retry forever or do we need to give up eventually and crash or something?

Reply:
Fixed it up. Not positive RemoteException is the best way to present connection failure...